### PR TITLE
RAD-86: Update exposure and program attributes information

### LINKED ISSUE #176: Add/Update exposure attribute information
_Issue [RAD-86](https://jira.stsci.edu/browse/RAD-86) was created on JIRA by [David Davis](https://jira.stsci.edu/secure/ViewProfile.jspa?name=ddavis):_

id: from "The exposure number for a given visit id" to something like "The exposure request number within a single activity is 5 characters long and is included in the observation ID" 

data_problem: Just add that it is set to "T" if a problem is detected. 

integration_time: "The effective time that the sensor has been exposed to the sky." add that this is corrected for dead time.

MA table --> multi-accumulation table

### PR DESCRIPTION
**Pull request by PaulHuwe**
Resolves [RAD-86](https://jira.stsci.edu/browse/RAD-86)

Closes #176

This PR updates exposure and program attributes information

## Tasks
- [ ] Update or add relevant `rad` tests.
- [ ] Update relevant docstrings and / or `docs/` page.
- [x] Does this PR change any schema files?
  - [x] Schema changes were discussed at RAD Review Board meeting.
- [x] Does this PR change any API used downstream? (If not, label with `no-changelog-entry-needed`.)
  - [ ] Write news fragment(s) in `changes/`: `echo "changed something" > changes/<PR#>.<changetype>.rst` (see below for change types).
  - [ ] Start a `romancal` regression test (https://github.com/spacetelescope/RegressionTests/actions/workflows/romancal.yml) with this branch installed (`"git+https://github.com/<fork>/rad@<branch>"`).
  - [ ] Update relevant `roman_datamodels` utilities and tests.

<details><summary>News fragment change types:</summary>

- ``changes/<PR#>.feature.rst``: new feature
- ``changes/<PR#>.bugfix.rst``: fixes an issue
- ``changes/<PR#>.doc.rst``: documentation change
- ``changes/<PR#>.removal.rst``: deprecation or removal of public API
- ``changes/<PR#>.misc.rst``: infrastructure or miscellaneous change
</details

### CODE REVIEW

File: src/rad/resources/schemas/exposure-1.0.0.yaml [hunk 1]
```diff
@@ -31,74 +37,76 @@ properties:
...
     archive_catalog:
       datatype: int
-      destination: [WFIExposure.exposure_frame_divisor, GuideWindow.exposure_frame_divisor, WFICommon.exposure_frame_divisor]
-  groupgap:
-    title: Number of Frames Dropped Between Resultants
-    description:
-      The number of reads that are dropped, or not used to calculate a
-      resultant.
-    type: integer
+      destination: [WFIExposure.exposure_ngroups, GuideWindow.exposure_ngroups, WFICommon.exposure_ngroups]
```
Comment (kdupriestsci) [inline src/rad/resources/schemas/exposure-1.0.0.yaml:90]: Do you want to rename the archive column name to match the keyword name?

**Comment by PaulHuwe** [comment]
Updated to address Issue #183

File: src/rad/resources/schemas/exposure-1.0.0.yaml (hunk 1 above)
Comment (schlafly) [reply src/rad/resources/schemas/exposure-1.0.0.yaml:90]: Yes; I don't think we want the "group" terminology showing up for Roman.

Comment (PaulHuwe) [reply src/rad/resources/schemas/exposure-1.0.0.yaml:90]: db pointing fixed

**Pull request merged**